TombEngine/Specific/Input/Input.cpp

```diff
@@ -88,30 +86,41 @@ namespace TEN::Input
 	int DbInput = 0;
 	int TrInput = 0;
 
-	auto DefaultBindings = std::vector<int>
-	{
-		KC_UP, KC_DOWN, KC_LEFT, KC_RIGHT, KC_PERIOD, KC_SLASH, KC_RSHIFT, KC_RMENU, KC_RCONTROL, KC_SPACE, KC_COMMA, KC_NUMPAD0, KC_END, KC_ESCAPE, KC_P, KC_PGUP, KC_PGDOWN,
-		/*KC_RCONTROL, KC_DOWN, KC_SLASH, KC_RSHIFT, KC_RMENU, KC_SPACE,*/ // TODO: Dedicated vehicle actions.
-		KC_F5, KC_F6, KC_RETURN, KC_ESCAPE, KC_NUMPAD0
+	// Rows:
+	// 1. General actions
+	// 2. Vehicle actions (TODO)
+	// 3. Quick actions
+	// 4. Menu controls
+	const auto DefaultBindings = std::vector<int>
+	{
+		KC_UP, KC_DOWN, KC_LEFT, KC_RIGHT, KC_DELETE, KC_PGDOWN, KC_RCONTROL, KC_RSHIFT, KC_SLASH, KC_PERIOD, KC_RMENU, KC_END, KC_SPACE, KC_NUMPAD0,
+		/*KC_RCONTROL, KC_DOWN, KC_SLASH, KC_RSHIFT, KC_RMENU, KC_SPACE,*/
```

Comment:
Probably it's time to get rid of this commented block since it makes no sense anymore because of new actions added to the end of the binding list.

Reply:
This is preparation for bindable vehicle actions which will come in a following PR. Currently I have everything stashed it.

Reply:
OK, I see!